SPECS/librdkafka/librdkafka.spec

```diff
@@ -0,0 +1,110 @@
+%global debug_package %{nil}
+%define soname 1
+%define _source_payload w9.gzdio
+%define _binary_payload w9.gzdio
+Summary:        The Apache Kafka C library
+Name:           librdkafka
+Version:        1.4.0
+Release:        1%{?dist}
+License:        BSD-2-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Libraries/C and C++
+URL:            https://github.com/edenhill/librdkafka
+Source0:        librdkafka-%{version}.tar.gz
+BuildRequires:  cyrus-sasl
+BuildRequires:  gcc >= 4.1
+BuildRequires:  libstdc++-devel
+BuildRequires:  openssl-devel
+BuildRequires:  python2
+BuildRequires:  zlib-devel
+
+%description
+librdkafka is the C/C++ client library implementation of the Apache Kafka protocol, containing both Producer and Consumer support.
+
+%package -n     %{name}%{soname}
+Summary:        The Apache Kafka C library
+Group:          Development/Libraries/C and C++
+Requires:       cyrus-sasl
+Requires:       libstdc++
+Requires:       openssl
+Requires:       zlib
+
+%description -n %{name}%{soname}
+librdkafka is the C/C++ client library implementation of the Apache Kafka protocol, containing both Producer and Consumer support.
+
+%package -n     %{name}-devel
+Summary:        The Apache Kafka C library (Development Environment)
+Group:          Development/Libraries/C and C++
+Requires:       %{name}%{soname} = %{version}
+
+%description -n %{name}-devel
+librdkafka is the C/C++ client library implementation of the Apache Kafka protocol, containing both Producer and Consumer support.
+This package contains headers and libraries required to build applications
+using librdkafka.
+
+%prep
+%setup -q
+
+# --install-deps will install missing dependencies that are not available
+# through BuildRequires, such as libzstd, which will be linked statically.
+%configure --install-deps --disable-lz4-ext
+
+%build
+cat config.log
+make
+examples/rdkafka_example -X builtin.features
+
+%install
+DESTDIR=%{buildroot} make install
+
+%clean
+rm -rf %{buildroot}
+
+%post   -n %{name}%{soname} -p /sbin/ldconfig
+%postun -n %{name}%{soname} -p /sbin/ldconfig
+
+%files -n %{name}%{soname}
+%defattr(444,root,root)
+%{_libdir}/librdkafka.so.%{soname}
+%{_libdir}/librdkafka++.so.%{soname}
+%defattr(-,root,root)
+%doc %{_docdir}/librdkafka/README.md
+%doc %{_docdir}/librdkafka/CONFIGURATION.md
+%doc %{_docdir}/librdkafka/INTRODUCTION.md
+%doc %{_docdir}/librdkafka/STATISTICS.md
+%license %{_docdir}/librdkafka/LICENSE
+%doc %{_docdir}/librdkafka/LICENSES.txt
+
+%defattr(-,root,root)
+#%{_bindir}/rdkafka_example
```

Comment:
Any particular reason why these are commented out from the final package?

Reply:
These are imported from the spec file in the librdkafka repo. Will remove them since they are not needed.

Reply:
Resolved

Reply:
Resolved